packages/cypress/src/plugins/plugin.spec.ts

```diff
@@ -157,7 +158,8 @@ describe('@nx/cypress/plugin', () => {
     `);
   });
 
-  it('should use ciDevServerTarget to create additional configurations', () => {
+  // TODO(Colum): Enable this
+  xit('should use ciDevServerTarget to create additional configurations', async () => {
```

Comment:
This test fails because of the following line in `setupWorkspaceContext` which doesn't play ball with `tempFs` 
https://github.com/nrwl/nx/blob/2374d8eaba89f1f82e789cc14b62230c2393862d/packages/nx/src/utils/workspace-context.ts#L9

It tries to require it from the tempFS's temporary directory, and it doesn't exist.